internal/util/render/executor.go

```diff
@@ -39,7 +39,10 @@ import (
 	"sigs.k8s.io/kustomize/kyaml/yaml"
 )
 
-var errAllowedExecNotSpecified = fmt.Errorf("must run with `--allow-exec` option to allow running function binaries")
+var (
+	errAllowedExecNotSpecified = fmt.Errorf("must run with `--allow-exec` option to allow running function binaries")
+	errAllowedWasmNotSpecified = fmt.Errorf("must run with `--allow-wasm` option to allow running wasm functions")
```

Comment:
Same with this error message, it should probably specify that this feature is alpha

Reply:
This is no longer needed. Removed now.